lib/tasks/say.js

```diff
@@ -210,7 +211,22 @@ class TaskSay extends TtsTask {
 
     await super.exec(cs);
     this.ep = ep;
+    if (cs.synthesizer) {
```

Comment:
shouldnt this be done in the superclass as well?

Reply:
Hi @davehorton  @rammohan-y 

I see that _synthesizeWithSpecificVendor is implemented in 2 places tts-task and say task and they are doing exactly the same excep som logging and notification that tts-task doing better.

I would suggest to migrate the _synthesizeWithSpecificVendor in say.js to tts-task and we have only one place to handle thing.

Reply:
@rammohan-y I agree with @xquanluu's point above, can you make that change?

Reply:
@davehorton moved _synthesizeWithSpecificVendor method from say.js to tts-task.js

Reply:
> Thanks, please do a thorough regression test, as this is a big change

The following were testing for both say and gather  + say

config contains synthesizer config, say verb doesn't contain synthesizer: synthesizer config from config is taken
config does not contain sythesizer config, verb contains synthesizer: synthesizer config from verb is taken
both config and verb has synthesizer options: synthesizer config from verb is taken
both config and verb does not contain synthesizer options: system took what is configured in the application

Reply:
Tested for playht, microsoft
Tested with disableTtsCache true